inference-engine/src/preprocessing/ie_preprocess_gapi_kernels_impl.hpp

```diff
@@ -117,7 +117,12 @@ template<typename DST, typename SRC> static inline DST checked_cast(SRC x) {
 }
 
 static inline Q8_8 mulas(Q0_16 a, U8   s) { return static_cast<Q8_8>((a * s) >>  8); }
-static inline Q8_8 mulaw(Q0_16 a, Q8_8 w) { return static_cast<Q8_8>((a * w) >> 16); }
+static inline Q8_8 mulaw(Q0_16 a, Q8_8 w) {
+   uint32_t mul = a;
+   mul *= w;
+   uint16_t res = static_cast<Q8_8>(mul >> 16);
```

Comment:
Also didn't get how to `uint16_t` match to `Q8_8` ? 
What does actually `Q8_8` means ?

Reply:
`#define Q8_8 uint16_t`